ports/servo/main.rs

```diff
@@ -142,7 +143,9 @@ fn main() {
     // Our wrapper around `Browser` that also implements some
     // callbacks required by the glutin window implementation.
     let mut browser = BrowserWrapper {
-        browser: Browser::new(window.clone()),
+        browser: Browser::new(window.clone(),
+                              opts::get().url.clone()
```

Comment:
opts.url checks for the PREFS shell.homepage. Shouldn't we remove that from the opts::from_cmdline_args and move it here instead?